packages/venia-concept/src/components/Navigation/categoryTree.js

```diff
@@ -54,6 +54,9 @@ class Tree extends Component {
                     });
 
                     const leaves = children.map(node => {
+                        if (!node.include_in_menu) {
```

Comment:
This type of boolean check will coerce `undefined` and cause unexpected behavior for anyone consuming this component still using a `2.3.0` backend (fun fact `!undefined === true`). I'm 99% sure our next release will be a major, but just in case let's fix this. 
```suggestion
                        if (node.include_in_menu === false) {
```

Reply:
I've fixed this condition as per your suggestion. Please review it